tests/Behat/Content/Venue/DefaultVenueContext.php

```diff
@@ -27,10 +27,17 @@ public function alterVenueFields(BeforeParseEntityFieldsScope $scope): void {
       'Address' => 'oe_address',
       'Capacity' => 'oe_capacity',
       'Room' => 'oe_room',
+      'Published' => 'status',
     ];
 
     foreach ($scope->getFields() as $key => $value) {
       if (isset($mapping[$key])) {
+        if ($key === 'Published') {
+          $scope->addFields([
+            $mapping[$key] => (int) ($value === 'Yes'),
+          ])->removeField($key);
+          continue;
+        }
```

Comment:
Let's use a switch here as we did in `EventContentContext` so we keep consistency.